Cargo.toml

```diff
@@ -15,11 +15,12 @@ name = "bitcoin_hashes"
 path = "src/lib.rs"
 
 [features]
-default = [ "std" ]
+default = ["std"]
 std = []
-# The no-std feature enables core2 and alloc.
+# The no-std feature enables core2 and the alloc feature adds the alloc crate to that.
```

Comment:
I'm not sure if that's the most clear way to say that the feature *does not* include `alloc`.